src/pages/Home/Home.jsx

```diff
@@ -1,4 +1,5 @@
 import Layout from "../../components/Layout/Layout";
+import UserProfileCard from "../../components/UserProfileCard/UserProfileCard";
```

Comment:
Although you're working on it and it needs to be rendered to work on, when making a pull request only include items that need to be rendered at that time and place. I will go over this in the E.O.D. chat.